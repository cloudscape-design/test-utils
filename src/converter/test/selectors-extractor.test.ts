// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
// SPDX-License-Identifier: Apache-2.0
import { test, expect } from 'vitest';
import path from 'path';
import { extractTestSelectorsUtil } from '../index';

const srcPath = path.join(__dirname, 'inputs', 'selectors-extractor', 'src');
const libPath = path.join(__dirname, 'inputs', 'selectors-extractor', 'lib', 'components');

test('selectors-extractor', () => {
  expect(extractTestSelectorsUtil({ srcPath, libPath })).toMatchSnapshot();
});
