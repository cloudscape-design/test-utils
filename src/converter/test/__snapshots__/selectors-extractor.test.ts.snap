// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`selectors-extractor 1`] = `
{
  "alert": [
    "awsui-alert-special",
    "awsui_action-button_mx3cw",
    "awsui_action-slot_mx3cw",
    "awsui_alert_mx3cw",
    "awsui_content_mx3cw",
    "awsui_dismiss-button_mx3cw",
    "awsui_header_mx3cw",
    "awsui_root_mx3cw",
  ],
}
`;
